cozy.example.yaml

```diff
@@ -235,31 +235,36 @@ mail:
       password: {{.Env.COZY_BETA_MAIL_PASSWORD}}
 
 # campaign mail service parameters for sending campaign emails via SMTP
+# If campaign_mail.host is empty, the default mail config will be used.
 campaign_mail:
-  # mail smtp host - flags: --mail-host
+  # SMTP server host
+  # Defaults to empty string
   host: smtp.home
-  # mail smtp port - flags: --mail-port
+  # SMTP server port
+  # Defaults to 0
```

Comment:
25 should be the default.

https://github.com/spf13/viper?tab=readme-ov-file#establishing-defaults